helm/templates/zookeeper.yaml

```diff
@@ -20,14 +21,35 @@ spec:
       name: "leader-election"
 
 ---
-apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.zookeeper.name | quote }}
+  namespace: {{ .Release.Namespace | quote }}
+data:
+  zoo.cfg: |
+    tickTime=2000
+    clientPort={{ .Values.zookeeper.port }}
+    initLimit=5
+    syncLimit=2
+    dataDir=/data
+    dataLogDir=/datalog
```

Comment:
I wonder if these values can be moved to values.yaml. If they can, it's good to have all configuration variables in a single place.

Reply:
Good point -- thanks!  Done.